06_PRC_Handson_User-Training.md

```diff
@@ -393,15 +439,22 @@ You can read a json file on your local pc and attach these keys and values as me
 
 ## How to make queries
 
-The PRC offers different query options that you can use based on your need. You may use these queries in your script to easily manage your research data.
+Different attributes of your objects, including the metadata, can be used to perform queries on your research data. This can be done with `session.query()` and a number of PRC classes imported from the `irods.models` module.
```

Comment:
I would say something like 'can be used when performing queries on your research data'. It's not these attributes that perform the queries themselves.